src/Compilers/VisualBasic/Portable/Lowering/LocalRewriter/LocalRewriter_Conversion.vb

```diff
@@ -1226,6 +1226,39 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Dim result As BoundExpression = node
             Dim member As WellKnownMember
 
+            Dim operand = node.Operand
+
+            If operand.Kind = BoundKind.Call AndAlso underlyingTypeTo.IsIntegralType() Then
```

Comment:
>If operand.Kind = BoundKind.Call AndAlso underlyingTypeTo.IsIntegralType() Then [](http://example.com/codeflow?start=12&length=79)

Despite the fact that legacy implementation doesn't have any comments, consider adding a comment explaining lowering strategy that we are trying to implement with the new code.

Reply:
I will add one later.